### DISCUSSION TITLE
is there any possibility to sniff packets of docker network in cli way ?

### DISCUSSION THREAD

**Discussion by Greatz08** [GyulyVGC/sniffnet#571] [General]
i wanted cli solution is there anyway with this project or can you recommend something for it :-))

**Comment by GyulyVGC** [comment]
Hi @HakaishinShwet, unfortunately there isn't a way to use the current Sniffnet version as a CLI.
As discussed in #29, after v0.5.0 I preferred to completely focus on the GUI development.
You can still sniff Docker networks by selecting the Docker virtual interfaces on Linux, but only via the GUI.
